packages/big-design-icons/src/flags/components/BZFlagIcon.tsx

```diff
@@ -12,7 +12,13 @@ const FlagIcon: React.FC<FlagIconProps & PrivateIconProps> = ({ svgRef, title =
   const titleId = title ? props.titleId || uniqueTitleId : undefined;
 
   return (
-    <svg viewBox="0 0 640 480" ref={svgRef} aria-labelledby={titleId} {...props}>
+    <svg
+      xmlnsXlink="http://www.w3.org/1999/xlink"
```

Comment:
This icon does have some `xlink:href` which I believe require the `xmlnsXlink` attribute to be present. It seems like the new version of `svgo` takes care of this